streamlit/GroMoPo_app.py

```diff
@@ -22,4 +24,11 @@
                 " questions, resources and groundwater models to the source code")
 st.sidebar.title("About")
 st.sidebar.info("This app is maintained and argued on by the GroMoPo mob")
-st.sidebar.image('pages/img/GroMoPo_logo_V1.png', caption=None, width=None, use_column_width=None, clamp=False, channels='RGB', output_format='auto')
+
+if platform.system() == 'Windows':
```

Comment:
This is where more operating systems could included in a list for local OS's. A better solution would be to somehow test if running on streamlit or locally.

if platform.system().lower() in ['windows','otherOS']:

Reply:
I added a comment to track this better.